src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InputFileResolverTests.cs

```diff
@@ -109,6 +109,45 @@ public void InitializeShouldFindFilesRecursively()
             result.ProjectContents.GetAllFiles().Count().ShouldBe(2);
         }
 
+        [Fact]
+        public void InitializeShouldUseBuildAlyzerResult()
```

Comment:
Does this test actually test if the initialize uses the analyzerResult? Both on filesystem and in the analyzerResult are 2 files present. So even if the analyzerResult is not used this test will still pass. Maybe we could have a difference here between filesystem and buildalyzer.